dpnp/backend/examples/example_bs_sycl.cpp

```diff
@@ -0,0 +1,97 @@
+#include <CL/sycl.hpp>
+#include <iostream>
+
+cl::sycl::event divide_sycl(double* price,
+                            double* strike,
+                            const size_t size,
+                            double* p_div_s,
+                            cl::sycl::queue q)
+{
+    double* P = sycl::malloc_shared<double>(size, q);
+    double* S = sycl::malloc_shared<double>(size, q);
```

Comment:
You don't need to allocate memory for `P` and `S`. It's already allocated for `price` and `strike` respectively. Let's just point to the memory with `double* P = price;`. It will look like a shortcut. Moreover you don't need to free the memory.